src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -55,8 +55,12 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach with TableDrivenPro
   val sfnServer = new WireMockServer(9004)
   val testOutputBucket = "outputBucket"
   val inputBucket = "inputBucket"
-  val packageAvailable: TREInput = TREInput(TREInputParameters("status", "TEST-REFERENCE", inputBucket, "test.tar.gz"))
-  val event: SQSEvent = createEvent(packageAvailable.asJson.printWith(Printer.noSpaces))
+  private def packageAvailable(s3Key: String): TREInput = TREInput(
```

Comment:
I think I told you this was the way to do it so sorry if I did but you don't need a second tar file. The metadata json for the tests comes from the `metadataJsonOpt` in the `stubAWSRequests` method so it doesn't need to be changed in the tar file. That means you can put these lines back the way they are on the main branch and then revert the changes from `event` to `event()`

Reply:
I've just seen the other PR that needs a different tar.gz file so it makes sense to leave this as you've done it. Ignore the comment on line 117 of LambdaTest as well.
You still don't need a new tar file for this PR though.